pkg/svcgen/params/params.go

```diff
@@ -0,0 +1,96 @@
+package params
+
+type Params struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Params should have comment or be unexported